spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,59 @@
+require 'beaker-rspec'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper
+
+install_ca_certs unless ENV['PUPPET_INSTALL_TYPE'] =~ /pe/i
+
+test_name "Installing Puppet Modules" do
+  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+  hosts.each do |host|
+    if host['platform'] =~ /windows/
+      on host, "mkdir -p #{host['distmoduledir']}/registry"
+      target = (on host, "echo #{host['distmoduledir']}/registry").raw_output.chomp
+
+      %w(lib manifests metadata.json).each do |file|
+        scp_to host, "#{proj_root}/#{file}", target
+      end
+      on host, 'curl -k -o c:/puppetlabs-stdlib-4.6.0.tar.gz https://forgeapi.puppetlabs.com/v3/files/puppetlabs-stdlib-4.6.0.tar.gz'
```

Comment:
I wonder if there are module_install_helpers for beaker we could use here instead?

e.g. https://github.com/puppetlabs/beaker-module_install_helper#install_module_dependencies_on

Side note - 4.6.0 is a REALLY old version

Reply:
Yeah, I was wondering the same - seems like there should be a helper for these bits.

Reply:
I can add this.